dev/protos/update.sh

```diff
@@ -77,31 +77,33 @@ cp protobuf/src/google/protobuf/{any,empty,field_mask,struct,timestamp,wrappers}
 # Generate the Protobuf typings
 PBJS_ARGS=( --proto_path=. \
   --js_out=import_style=commonjs,binary:library \
-  --target=static \
+  --target=static-module \
   --no-create \
   --no-encode \
   --no-decode \
   --no-verify \
-  --no-convert \
   --no-delimited \
   --force-enum-string \
   --force-number)
       
 "${PBJS}" "${PBJS_ARGS[@]}" -o firestore_v1_proto_api.js \
+  -r firestore_v1 \
   "${PROTOS_DIR}/google/firestore/v1/*.proto" \
   "${PROTOS_DIR}/google/protobuf/*.proto" "${PROTOS_DIR}/google/type/*.proto" \
   "${PROTOS_DIR}/google/rpc/*.proto" "${PROTOS_DIR}/google/api/*.proto" \
   "${PROTOS_DIR}/google/longrunning/*.proto"
 "${PBTS}" -o firestore_v1_proto_api.d.ts firestore_v1_proto_api.js
 
 "${PBJS}" "${PBJS_ARGS[@]}" -o firestore_admin_v1_proto_api.js \
+  -r firestore_admin_v1 \
   "${PROTOS_DIR}/google/firestore/admin/v1/*.proto" \
   "${PROTOS_DIR}/google/protobuf/*.proto" "${PROTOS_DIR}/google/type/*.proto" \
   "${PROTOS_DIR}/google/rpc/*.proto" "${PROTOS_DIR}/google/api/*.proto" \
   "${PROTOS_DIR}/google/longrunning/*.proto"
 "${PBTS}" -o firestore_admin_v1_proto_api.d.ts firestore_admin_v1_proto_api.js
 
 "${PBJS}" "${PBJS_ARGS[@]}" -o firestore_v1beta1_proto_api.js \
+  -r firestore_v1beta1_v1 \
```

Comment:
Does not really affect anything but the name is weird. Maybe just `firestore_v1beta1`?

Reply:
Fixed